src/gui/optionsdlg.ui

```diff
@@ -2976,7 +2976,27 @@ Use ';' to split multiple entries. Can use wildcard '*'.</string>
                   <item row="4" column="0" colspan="2">
                    <widget class="QCheckBox" name="checkBypassLocalAuth">
                     <property name="text">
-                     <string>Bypass authentication for localhost</string>
+                     <string>Bypass authentication for clients on localhost</string>
+                    </property>
+                   </widget>
+                  </item>
+                  <item row="5" column="0" colspan="2">
+                   <widget class="QCheckBox" name="checkBypassAuthIPSubnet">
+                    <property name="text">
+                     <string>Bypass authentication for clients in the IP subnet whitelist</string>
```

Comment:
Bypass authentication for clients in whitelisted IP subnets?

Reply:
@evsh:  That can work, though both are correct as far as English grammar goes.

Reply:
My point was that the white list contains not clients but sub-nets.

Reply:
@evsh:  My apologies…I figured that out **after** I had already commented.  :{